src/content/reference/react-dom/unmountComponentAtNode.md

```diff
@@ -39,21 +39,21 @@ render(<App />, domNode);
 unmountComponentAtNode(domNode);
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lainnya di bawah ini.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `domNode`: A [DOM element.](https://developer.mozilla.org/en-US/docs/Web/API/Element) React will remove a mounted React component from this element.
+* `domNode`: Sebuah [elemen DOM.](https://developer.mozilla.org/en-US/docs/Web/API/Element) React akan menghapus komponen React yang terpasang dari elemen ini.
 
-#### Returns {/*returns*/}
+#### Pengembalian {/*returns*/}
 
-`unmountComponentAtNode` returns `true` if a component was unmounted and `false` otherwise.
+`unmountComponentAtNode` mengembalikan `true` jika sebuah komponen telah dicopot dan `false` jika tidak.
 
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-Call `unmountComponentAtNode` to remove a <CodeStep step={1}>mounted React component</CodeStep> from a <CodeStep step={2}>browser DOM node</CodeStep> and clean up its event handlers and state.
+Panggil `unmountComponentAtNode` untuk menghapus <CodeStep step={1}>komponen React yang sudah terpasang</CodeStep> dari <CodeStep step={2}>simpul DOM peramban</CodeStep> dan bersihkan *event handler* dan *state*-nya.
```

Comment:
```suggestion
Panggil `unmountComponentAtNode` untuk menghapus <CodeStep step={1}>komponen React yang sudah terpasang</CodeStep> dari <CodeStep step={2}>simpul DOM peramban</CodeStep> dan bersihkan *event handlers* beserta *state*-nya.
```

1. Retain the `handlers` plural form
2. Give some variations to prevent too many `dan` conjunction occurrences.

Reply:
@zainfathoni I think we should convert all plural forms to singular in Indonesian, since there's no such thing as plural forms in Indonesian.

Reply:
@zainfathoni @resir014

 Between these:

A. *event handlers*
B. banyak *event handler*

Either choose by voting, or ask Ivan Lanin on Twitter/X 😁

Reply:
[Asking](https://twitter.com/heychez14/status/1686542531029499904) Uda Ivan… I'll let you know if there's an update. Thank you!

Reply:
To unblock this, you may ignore my comments on retaining the plural forms.
So @chez14, please fix the other requested changes, and you should be good to go. Thanks!